test/interpreter/fixtures/fneg/exec.tjs

```diff
@@ -0,0 +1,36 @@
+const m = WebAssembly.instantiateFromSource(watfmodule);
+
+it('should negate positive input', () => {
+  const res = m.exports.neg(0.5);
+
+  assert.typeOf(res, 'number');
+  assert.equal(res, -0.5);
+});
+
+it('should negate negative input', () => {
+  const res = m.exports.neg(-12341234);
+
+  assert.typeOf(res, 'number');
+  assert.equal(res, 12341234);
```

Comment:
The result type should be a float?

Reply:
Not sure what you mean, integers are floats?

Reply:
I expected 12341234.0 but you are right.

Reply:
Ah yeah these are the same thing in JS